### DISCUSSION TITLE
Log-in

### DISCUSSION THREAD

**Discussion by HubeStyle** [ORelio/Minecraft-Console-Client#2584] [Q&A]
Hi, i was wondering if i've migrated my Minecraft account with Microsoft will it not work? I can't log in , using the migrated account and i don't remember my old normal password. What do i do?

**Comment by ReinforceZwei** [comment]
You should go contact Mojang if you forgot your account password.

**Reply by HubeStyle** [reply]
Even if its migrated? it'll still work?

**Reply by HubeStyle** [reply]
i know my microsoft password but it dont work with the console
